lib/generateNormals.js

```diff
@@ -14,10 +17,7 @@ module.exports = generateNormals;
  * @param gltf
  */
 function generateNormals(gltf) {
-    var primitives = getPrimitives(gltf);
-    var primitivesLength = primitives.length;
-    for (var i = 0; i < primitivesLength; i++) {
-        var primitive = primitives[i];
+    jp.apply(gltf, '$.meshes[*].primitives[*]', function (primitive) {
```

Comment:
I did a quick look around and didn't find much for existing performance benchmarks. I'll throw something together later this week. Might even make an interesting blog post.